### PR TITLE
Add and remove branch

### PR DESCRIPTION
**Pull request by 10-menachi**
Added:
- Add functionality
- Remove functionality
- Edit functionality

### CODE REVIEW

File: src/modules/deleteTask.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import getTasks from './getTasks';
+
+const deleteTask = (id) => {
+  let tasks = getTasks();
+  tasks = tasks.filter((task) => Number(task.id) !== Number(id));
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+export default deleteTask;
```
Comment (mikethreels) [inline src/modules/deleteTask.js]: - [ ] According to the requirements, Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. as you can see in the below print screen that is not the case right now.
afbeelding

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi @10-menachi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- great job saving to the local storage.

## Required Changes ♻️
- [ ] the link to the MIT license in the readme file is not working as expected, please make sure it is functioning as presented in [the readme template](https://github.com/microverseinc/readme-template) provided by Microverse.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/Task.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class Task {
+  constructor(id, name, done) {
+    this.id = id;
```
Comment (brytebee) [inline src/modules/Task.js]: - [ ] **Optional:** Attempt to start the id from 1.

Consider this code

```
this.id = id + 1;
```

File: src/modules/deleteTask.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import getTasks from './getTasks';
+
+const deleteTask = (id) => {
+  let tasks = getTasks();
+  tasks = tasks.filter((task) => Number(task.id) !== Number(id));
+  // update task IDs to avoid duplicates
+  tasks = tasks.map((task, index) => {
+    task.id = index;
+    return task;
+  });
```
Comment (brytebee) [inline src/modules/deleteTask.js:10]: While this works when the delete button is clicked, it doesn't work when the clear all button is used. Could you implement similar functionality for clear all?

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @10-menachi 👋, 

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by jerryowusu — APPROVED** [review]
Hi @10-menachi  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Delete function working fine :+1: 
Todo created successfully :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**